docs/diagrams/RedoSequenceDiagram.puml

```diff
@@ -0,0 +1,74 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "u:RedoCommand" as RedoCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant ":AddressBook" as AddressBook MODEL_COLOR
+end box
+[-> LogicManager : execute(redo)
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(redo)
+activate AddressBookParser
+
+create RedoCommand
+AddressBookParser -> RedoCommand
```

Comment:
Hello~~ to follow to our team's convention, you can include the :RedoCommand at the end of line 21 💯